gen/http/openapi3.yaml

```diff
@@ -47,6 +47,179 @@ paths:
       responses:
         "200":
           description: File downloaded
+  /codesets:
+    get:
+      tags:
+      - codeset
+      summary: list codeset
+      description: Retrieve information about Codesets registered in FuseML.
+      operationId: codeset#list
+      parameters:
+      - name: project
+        in: query
+        description: List only Codesets that belong to given project
+        allowEmptyValue: true
+        schema:
+          type: string
+          description: List only Codesets that belong to given project
+          example: mlflow-project-01
+        example: mlflow-project-01
+      - name: label
+        in: query
+        description: List only Codesets with matching label
+        allowEmptyValue: true
+        schema:
+          type: string
+          description: List only Codesets with matching label
+          example: mlflow
+        example: mlflow
+      responses:
+        "200":
+          description: OK response.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Codeset'
+                description: Return all registered Codesets matching the query.
+                example:
+                - description: My first MLFlow application with FuseML
+                  label: mlflow
+                  name: mlflow-app-01
+                  project: mlflow-project-01
+                - description: My first MLFlow application with FuseML
+                  label: mlflow
+                  name: mlflow-app-01
+                  project: mlflow-project-01
+              example:
+              - description: My first MLFlow application with FuseML
+                label: mlflow
+                name: mlflow-app-01
+                project: mlflow-project-01
+              - description: My first MLFlow application with FuseML
+                label: mlflow
+                name: mlflow-app-01
+                project: mlflow-project-01
+        "404":
+          description: Not Found response.
+          content:
+            application/vnd.goa.error:
+              schema:
+                $ref: '#/components/schemas/Error'
+              example:
+                id: 3F1FKVRR
+                message: Value of ID must be an integer
+                name: bad_request
+    post:
+      tags:
+      - codeset
+      summary: register codeset
+      description: Register a Codeset with the FuseML codeset store.
+      operationId: codeset#register
+      parameters:
+      - name: location
```

Comment:
`location` is still showing in the REST API as a POST op parameter. I think the only way to avoid this is to leave it out of the goa design spec completely and add it manually only as a CLI command line parameter, given that the `location` only makes sense on the client side.